Nodesets/Opc.Ua.WoT.NodeSet.xml

```diff
@@ -0,0 +1,1242 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is an intermediate file from model compiler and can be removed.

Reply:
Will remove.